### ISSUE TITLE
Markowitz example not working

### ISSUE THREAD

**Issue by stephane-caron** [SCIP-Interfaces/PySCIPOpt#46]
Hi,

The example ``markowitz_soco.py`` raises an exception:
```python
/usr/local/lib/python2.7/dist-packages/pyscipopt/scip.so in pyscipopt.scip.Model.setObjective (pyscipopt/scip.c:38918)()

ValueError: Nonlinear objective functions are not supported!
```

Were nonlinear objectives working before and not any more? Or available in SCIP but not in PySCIPOpt?

Anyhow, the example should be updated I guess.

Best,

**Comment by fserra** [comment]
Oh I am sorry, this example shouldn't be in the `finished` directory.

SCIP doesn't allow for nonlinear objective. It should be reformulated as `min t; t >= nonlinear_obj, etc`

I will take care of this 

Thanks!

**Comment by fserra** [comment]
fixed with ed64699
